devtools/conda-envs/test.yaml

```
@@ -4,7 +4,7 @@ channels:
 dependencies:
   # Core
   - python
-  - cython =0.29.36
+  - cython
```

I checked the logs and 3.x is pulled down, but just to be safe (and explicit about dependencies)
```suggestion
  - cython =3
```

---

Done.

---

Ah, this actually doesn't work and it should be `~3`.